app/frontend/components/pages/login/mnemonicAuth.js

```diff
@@ -59,7 +64,7 @@ const LoadByMenmonicSection = ({
       'a',
       {
         class: 'intro-link fade-in-up',
-        onClick: openGenerateMnemonicDialog,
+        onMouseDown: (e) => isLeftClick(e, openGenerateMnemonicDialog),
```

Comment:
this is to prevent mnemonic field onBlur happen before onClick (validator will show err, which moves layout, so click might end outside the button)